CMakeLists.txt

```diff
@@ -15,6 +15,7 @@ set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
 set(CMAKE_Swift_MODULE_DIRECTORY ${CMAKE_BINARY_DIR}/swift)
 
 include(CTest)
+include(GNUInstallDirs)
```

Comment:
`GNUInstallDirs` requires either C or C++ be enabled to work correctly.

```
CMake Warning (dev) at ...share/cmake-3.30/Modules/GNUInstallDirs.cmake:253 (message):
  Unable to determine default CMAKE_INSTALL_LIBDIR directory because no
  target architecture is known.  Please enable at least one language before
  including GNUInstallDirs.
Call Stack (most recent call first):
  CMakeLists.txt:18 (include)
This warning is for project developers.  Use -Wno-dev to suppress it.
```

Reply:
Thank you, I enabled C.